js/WSConnection.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * WSConnection Component
+ * -------------------------------------------
+ * Custom Element describing connections between worksheets
+ */
+import LeaderLine from 'leader-line';
+
+class WSConnection extends HTMLElement {
+    constructor(){
+        super();
+
+        // Bound component methods
+        this.updateLeaderLine = this.updateLeaderLine.bind(this);
+        this.updateLinkedSheet = this.updateLinkedSheet.bind(this);
+        this.onWorksheetMoved = this.onWorksheetMoved.bind(this);
+    }
+
+    connectedCallback(){
+        if(this.isConnected){
+            this.updateLinkedSheet(
+                "",
+                this.getAttribute("source")
+            );
+            this.updateLinkedSheet(
+                "",
+                this.getAttribute("destination")
```

Comment:
up until now we call this attribute "target" (source -> target). It's not so important what we call but there is already some code out there - css class toggles, and how the instructions are processed - which depends on it.

Reply:
Fixed in https://github.com/dkrasner/worksheet/pull/23/commits/e41dd7a47f28cd27e3d7ac133295ebd11b6f3ed1